client/packages/system/src/IndicatorsDemographics/DetailView/IndicatorsDemographics.tsx

```diff
@@ -209,7 +190,14 @@ const IndicatorsDemographicsComponent: FC = () => {
               : await updateIndicator(indicator);
           }
         })
-      ).then(() => invalidateQueries());
+      )
+        .then(async () => {
+          if (headerDraft !== undefined)
+            await upsertProjection(mapProjection(headerDraft));
```

Comment:
Because you're doing this inside the loop, projections aren't saved if you have no indicators... also seems unnecessary to re-upsert the projections for every indicator - maybe do before the loop?

Reply:
cool - will do 🙏